src/BSON/Binary.c

```diff
@@ -168,7 +205,11 @@ HashTable *php_phongo_binary_get_debug_info(zval *object, int *is_temp TSRMLS_DC
 	*is_temp = 1;
 	array_init_size(&retval, 2);
 
+#if PHP_VERSION_ID >= 70000
+        add_assoc_stringl_ex(&retval, ZEND_STRS("data"), intern->data, intern->data_len);
+#else
 	add_assoc_stringl_ex(&retval, ZEND_STRS("data"), intern->data, intern->data_len, 1);
```

Comment:
I change my mind on this one when I realized we already had similar macro for dealing with int64 (ADD_INDEX_INT64()/ADD_ASSOC_INT64()), so I've added ADD_ASSOC_STRING().
See https://jira.mongodb.org/browse/PHPC-385